apps/tailwind-components/components/table/CellEMX2.vue

```diff
@@ -12,7 +12,11 @@
     />
 
     <ValueString
-      v-else-if="metadata.columnType === 'STRING'"
+      v-else-if="
+        metadata.columnType === 'STRING' ||
+        metadata.columnType === 'DATE' ||
+        metadata.columnType === 'DATETIME'
+      "
       :metadata="metadata"
       :data="data"
```

Comment:
I'm defaulting to `ValueString`. If these need to be displayed as a different type, please let me know.